package.json

```diff
@@ -53,6 +53,7 @@
     "plotly.js-gl2d-dist": "2.14.0",
     "printj": "1.3.1",
     "resolve-url-loader": "5.0.0",
+    "sanitize-html": "^2.8.1",
```

Comment:
Can you remove the 🥕

Reply:
oh dang... thought I did, maybe I didn't push it. Roger Wilco.